generators/client/templates/angular/src/test/javascript/spec/app/account/password/_password.component.spec.ts

```diff
@@ -52,11 +52,8 @@ describe('Component Tests', () => {
                     <%_ } _%>
                     Password
                 ]
-            }).overrideComponent(PasswordComponent, {
```

Comment:
WTF so they just suddenly decided to change this behaviour. @sendilkumarn You are good at finding the hidden issues man :clap: